db/schema.rb

```diff
@@ -0,0 +1,51 @@
+# rubocop:disable Metrics/BlockLength
+ActiveRecord::Schema[7.0].define(version: 20_221_209_131_446) do
+  # These are extensions that must be enabled in order to support this database schema
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.string 'text'
```

Comment:
Please use data type `text` instead of `string` for `text` attribute in `comments` table as `text` data type can store more characters compared to string.